senderscore.py

```diff
@@ -3,25 +3,53 @@
 
 
 class SenderScore(BotPlugin):
-    """Plugin to show the mail score of the given ip"""
+    """Plugin to show the mail score of the given address"""
 
     @botcmd
     def senderscore(self, msg, args):
-        """retrieve mail reputation usage !senderscore [IP]"""
-        return self.get_senderscores(args)
+        """retrieve mail reputation usage !senderscore [IP/Hostname]"""
+        if self.is_ip(args):
+            ipadress = args
+            score = self.get_senderscores(ipadress)
+            if score:
+                yield ("The reputation of {} is: {}".format(ipadress, score))
+            else:
+                yield ("Could'nt find a reputation for: {}".format(ipadress))
+        else:
+            hostname = args
+            ips = self.get_host_ips(hostname)
+            for ip in ips:
+                score = self.get_senderscores(ip)
```

Comment:
This is almost a copy of the previous code, but now ipadress is called ip, even though it's the same thing.

You could maybe even do something like
```
if self.is_ip(args):
             ip_addresses = [args]
else:
             ip_addresses = self.get_host_ips(args)
for  ip_address in ip_addresses:
```
and then keep the rest the same.